packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js

```diff
@@ -77,18 +79,110 @@ describe('Component > TranscribedLines', function () {
   })
 
   describe('with all lines', function () {
+    let consensusComponents, createMarkSpy, returnRefs, showConsensusStub
+
     before(function () {
       sinon.spy(React, 'createRef')
-      wrapper = shallow(<TranscribedLines lines={consensusLines} task={task} />)
+      createMarkSpy = sinon.spy(TranscribedLines.prototype, 'createMark')
+      showConsensusStub = sinon.stub(TranscribedLines.prototype, 'showConsensus')
```

Comment:
Why is this a stub and `createMark` a spy?

Reply:
If `showConsensus` is a spy and called as is, it triggers a state change, which causes issues with other "should call ConsensusLine callback with expected ref (on click/enter/space)". I'm not entirely sure why, the test error message includes "Invariant Violation: Calling PropTypes validators directly...", but I'm not sure why that's getting thrown.

Reply:
I don't know if it's related, but the Consensus Popup stories throw that same error. #1789.

Reply:
It means a prop type definition somewhere is being incorrectly called like `PropTypes.bool()` instead of `PropTypes.bool`, so that's what we should look for.

Reply:
I found that changing `StyledBox` to `Box` in `ConsensusPopup` fixed it, but couldn't figure out why the styled component would throw that error.

Reply:
It appears the `${List}` styling selector in ConsensusPopup was causing the issue, though I'm unsure exactly how/why, but I've edited the ConsensusPopup component to select `ul` instead.

Reply:
Changing `showConsensusStub` to a spy allows `showConsensus` to change component state, I think thereby triggering a rerender, which then changes the createRef spy returnValues (`returnRefs`) and breaks the tests.

Reply:
I see. That's makes sense after refamiliarizing myself of what is going on here.